docker-compose.yml

```diff
@@ -107,6 +107,24 @@ services:
     networks:
       - mds
 
+  ####################### NRIS_BACKEND Definition #######################
+  nris_backend:
+    restart: always
+    container_name: nris_python
+    build:
+      context: microservices/nris_api/backend
+      dockerfile: Dockerfile
+    ports:
+      - 5500:5500
+    volumes:
+      - ./microservices/nris_api/backend/:/app/
+    depends_on:
+      - flyway
+    # Export environment variables from .env into the container environment
+    entrypoint: ["bash", "-c", "grep -v '^#' .env | xargs ; flask run"]
```

Comment:
`entrypoint` can be a bash file, which might be easier to maintain than this command?

Is there a reason not to just COPY an .env file with the relevant args into the container and then `. .env` or `source .env` if necessary when initializing the container? That could all be managed in the Dockerfile instead of in the compose file

Reply:
hmmm that's an artifact from copy-paste... but it does smell, let me take a look.

Reply:
Any reason that we aren't using the `env-file` parameter for docker-compose? 
https://docs.docker.com/compose/compose-file/#env_file